coviu-calls.php

```diff
@@ -435,66 +435,32 @@ function delete_host(host_id) {
 		// Recover coviu
 		$coviu = new Coviu($options->api_key, $options->api_key_secret);
 
-		$current_user = wp_get_current_user();
-
-		if (current_user_can( 'edit_posts' )) {
-			$params = array();
-		} else {
-			$params = array(
-				'state' => $current_user->ID,
-			);
-		}
-
-		$sessions = $coviu->sessions->getSessions($params);
-		$sessions = $sessions['content'];
-
-		foreach ($sessions as $key => $session) {
-			$sessions[$key]['start_time'] = new DateTime($session['start_time']);
-			$sessions[$key]['end_time']   = new DateTime($session['end_time']);
-		}
-
-		function cmp_by_time($session1, $session2) {
-			return $session1['start_time'] < $session2['start_time'];
-		}
-		// sort by start_time
-		usort($sessions, 'cmp_by_time');
-
-		$upcoming_split_index = 0;
-		$active_sessions = [];
-		$now = new DateTime();
-
-		// remove current sesions from array
-		foreach ($sessions as $key => $session) {
-			if ( $now >= $session['start_time'] && $now <= $session['end_time']) {
-				array_push( $active_sessions, $session );
-				unset( $sessions[$key] );
-			}
-		}
-		array_values($sessions);
-
-		// determine split point in remaining sessions
-		foreach ($sessions as $key => $session) {
-			if ($now > $session['start_time']) break;
-			$upcoming_split_index++;
+		$params = array(
+			'page_size' => 10,
```

Comment:
move it into the past sessions retrieval function - I think we should probably retrieve all current and all future sessions